modules/virt-building-vm-containerdisk-image.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * virt//support/monitoring/virt-running-cluster-checkups.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-building-vm-containerdisk-image_{context}"]                                  
+= Building a container disk image for RHEL virtual machines
+
+You can build a custom RHEL 8 OS image in `qcow2` format and use it to build a container disk image for RHEL virtual machines (VMs). You can store the container disk image in a registry that is accessible from your cluster and specify the image location in the `spec.param.vmContainerDiskImage` attribute of the DPDK checkup config map. 
+
+Run the following procedure in the image builder VM to build a container disk image for RHEL 8.7 VMs with 2 CPU cores, 4 GiB RAM, and 20 GB of free space in the `/var` directory. 
+
+
+.Prerequisites
+* You have access to the image builder tool and its CLI (`composer-cli`).
+* You have verified that you can build a RHEL 8.7 image:
++
+[source,terminal]
+----
+# composer-cli distros list
+----
++
+[NOTE]
+====
+To run the `composer-cli` commands as non-root, add your user to the `weldr` or `root` groups:
+
+[source,terminal]
+----
+# usermod -a -G weldr user
+----
+[source,terminal]
+----
+$ newgrp weldr
+----
+====
+* You have installed the `virt-customize` tool.
```

Comment:
Please consider adding a link that explains how to install the `virt-customize` tool.
Alternatively, the command to install it on RHEL 8 is:
```bash
# dnf install libguestfs-tools
```